README.md

```diff
@@ -2,9 +2,16 @@
 This is an OpenShift plugin to ElasticSearch to:
 
 * Dynamically update the SearchGuard ACL based on a user's name
-* Transform kibana index requests to support multitenant deployments
+* Transform kibana index requests to support multitenant deployments for non-operations users.
 * Seed the Searchguard index `config`, `roles`, `rolesmapping`, and `actiongroups` types
 
+*Note:*
+Previous versions of this plugin created a Kibana profile for each user regardless of their role.
+As of v2.4.4.2, any users that has been given the OpenShift cluster-admin role will share the same
+Kibana index to store configurations and dashboards.  This change makes it possible to share dashboards
+between operation users.  It is highly recommended that operations teams establish agreements and
+naming conventions so users do not overwrite each others work.
```

Comment:
We also should dove tail this work with the unified dashboard work for non-ops users.